### ISSUE TITLE
Evaluate Flynn

### ISSUE THREAD

**Issue by fabiofumarola** [DataToKnowledge/wheretolive-backend#19]
We actually use dokku to run our services, but we need something that allow us to automate scaling and other stuff. We need to evaluate [flynn.io](https://flynn.io/)

**Comment by fabiofumarola** [comment]
Error during the installation test on local machine

```
6:46:34.882874 wait controller-wait
16:46:34.884612 add-app controller-inception
16:46:34.886617 add-app controller-inception error: POST http://100.100.11.4:80/apps: httpclient: unexpected status 401
Connection to 127.0.0.1 closed.
make[2]: *** [ssh-cmd] Error 1
make[1]: *** [bootstrap] Error 1
make: *** [init] Error 2
```

**Comment by fabiofumarola** [comment]
I have evaluated flynn there is not so much documentation.
We will consider the project in the future paying attention to:
1. documentation
2. supported buildsteps

The current version has  I have evaluated flynn there is not so much documentation.
We will consider the project in the future paying attention to:
1. documentation
2. supported buildsteps

The current version has an error when builded
